src/import/sst_importer.rs

```diff
@@ -185,6 +204,35 @@ impl ImportDir {
         }
         Ok(path)
     }
+
+    fn ingest(&self, meta: &SSTMeta, db: &DB) -> Result<()> {
+        let path = self.join(meta)?;
+        let cf = meta.get_cf_name();
+        prepare_sst_for_ingestion(&path.save, &path.clone)?;
+        validate_sst_for_ingestion(db, cf, &path.clone, meta.get_length(), meta.get_crc32())?;
+
+        let handle = get_cf_handle(db, cf)?;
+        let mut opts = IngestExternalFileOptions::new();
+        opts.move_files(true);
+        db.ingest_external_file_cf(handle, &opts, &[path.clone.to_str().unwrap()])?;
```

Comment:
When TiKV crash after sst have been ingested but before return Ok, whose responsibility it is to retry?